web-console/4.0-designs/status/status.md

```diff
@@ -1,5 +1,39 @@
 # Displaying Status
-- Status can be displayed in a number of ways depending on the use case and page content. When displaying status in the console, use the following design recommendations regarding the visual treatment.
+Status is primarily displayed on pages such as Summary and Details pages. Other pages status may appear on are Subscriptions and Search.
+
+Summary Page
+![nodes](img/nodes.png)
+
+Details Page
+![deployment](img/deployment.png)
 
 ## High Level Recommendations
-- ![status](img/status.png)
+Status can be displayed in a number of ways depending on the use case and page content. When displaying status in the console, use the following design recommendations regarding the visual treatment.
```

Comment:
when do/should you use the text only treatment? only if it doesn't match any of the examples?

Reply:
@serenamarie125 Agreed!